lib/pages/chat_details/chat_details_view.dart

```diff
@@ -29,7 +29,8 @@ class ChatDetailsView extends StatelessWidget {
       );
     }
 
-    controller.members!.removeWhere((u) => u.membership == Membership.leave);
+    controller.membersNotifier.value!
```

Comment:
Are you sure it's safe ? 
` controller.membersNotifier.value!`

Reply:
IMO: Move this func to controller, logic should not be handled directly in the view

Reply:
remove it, because it's not needed